finagle-base-http/src/test/scala/com/twitter/finagle/http/DefaultHeaderMapTest.scala

```diff
@@ -3,7 +3,7 @@ package com.twitter.finagle.http
 import org.scalacheck.Gen
 import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
 
-class DefaultHeaderMapTest extends AbstractHeaderMapTest with ScalaCheckDrivenPropertyChecks {
+class TrieHeaderMapTest extends AbstractHeaderMapTest with ScalaCheckDrivenPropertyChecks {
```

Comment:
let's make sure we're testing the Map-backed HeaderMap too

Reply:
Yes, that sounds good. Is there a reason why these tests aren't on `AbstractHeaderMapTest`? Should I duplicate them or move them there?